roofit/roofit/src/RooBernstein.cxx

```diff
@@ -216,6 +218,24 @@ RooSpan<double> RooBernstein::evaluateBatch(std::size_t begin, std::size_t batch
   return output;
 }
 
+
+RooSpan<double> RooBernstein::evaluateSpan(BatchHelpers::RunContext& evalData, const RooArgSet* normSet) const {
+  auto xData = _x->getValues(evalData, normSet);
+  size_t batchSize = xData.size();
+  if (batchSize < 2) {
+        return {};
+  }
+  
+  auto output = evalData.makeBatch(this, batchSize);
+  int nCoef = _coefList.size();
```

Comment:
`const int`?
Doesn't make a difference for the compiler, but might tell readers that this is only a temporary helper.